crates/m3/src/builder/expr.rs

```diff
@@ -11,6 +11,8 @@ use super::{column::Col, table::TableId};
 pub struct ZeroConstraint<F: Field> {
 	pub name: String,
 	pub expr: ArithCircuit<F>,
+	// TODO: tower_level
```

Comment:
TODO?

Reply:
Oh yeah, that was a TODO to ask a question here:

So `ArithCircuit` now has a `binary_tower_level`. I assume this is a different one, so I assume I should leave this one here, correct?